packages/api/src/command/usage/report-usage.ts

```diff
@@ -1,24 +1,20 @@
 import Axios from "axios";
+import { Product } from "../../domain/product";
 import { Config } from "../../shared/config";
 import { capture } from "../../shared/notifications";
 
-const axios = Axios.create({ timeout: 10_000 });
-
-export enum ApiTypes {
-  devices = "devices",
-  medical = "medical",
-}
```

Comment:
Moved to a [better domain representation](https://github.com/metriport/metriport/pull/820/files#diff-1e71c47aac36f66ab5a40ce15a14069849ff8a0845110f343342af78af838936R1), which is independent on usage reporting.